### DISCUSSION TITLE
implementation of multilingual

### DISCUSSION THREAD

**Discussion by ivictor777** [Laravel-Backpack/community-forum#1047] [Q&A (Help)]
Hello! How multilingual support is implemented here?
https://demo.backpackforlaravel.com/admin/product/210/edit?_locale=ro

**Comment by karandatwani92** [comment]
Hey @ivictor777 

Here is the guide:
https://backpackforlaravel.com/docs/6.x/crud-operation-update#translatable-models-and-multi-language-cruds

Here is the demo's implementation for products CRUD:
https://github.com/Laravel-Backpack/demo/blob/main/app/Models/Product.php
